docs/clusterfuzzlite.md

```diff
@@ -7,44 +7,40 @@ permalink: /
 ---
 
 # ClusterFuzzLite
+ClusterFuzzLite is a continuous fuzzing solution that runs as part of CI/CD workflows to find vulnerabilities faster than ever before. 
+With just a few lines of code, GitHub users can integrate ClusterFuzzLite into their workflow and fuzz pull requests to catch bugs before they are committed, enhancing the overall security of the software supply chain.
 
-ClusterFuzzLite makes [fuzzing] part of [Continuous Integration (CI)].
 ClusterFuzzLite is based on [ClusterFuzz].
 
-Fuzzing is a highly effective technique for finding bugs in software.
-ClusterFuzzLite makes your code more secure by fuzzing changes before they even
-enter the codebase.
+## Support
+- C, C++, Java (and other JVM-based languages), Go, Python, Rust, and Swift
+- [GitHub Actions], [Google Cloud Build], and [Prow] (with more CI systems in progress)
+- [Extending support to other CI systems] is easy
 
-ClusterFuzzLite supports [GitHub Actions], [Google Cloud Build], and [Prow], so
-getting the benefits of continuous fuzzing is simple if you can use these CI
-systems.
-Suppport for more CI systems is in-progress and [extending support to other CI
-systems] is easy.
-ClusterFuzzLite supports programming languages and runtimes beyond C and C++,
-including Java (and other JVM-based languages), Go, Python, Rust, and Swift.
+## Features
 
-ClusterFuzzLite offers many useful features, including:
 - Quickly fuzzing code changes (pull requests) before they land and allowing
-   you to download the crashing testcases.
+   you to download the crashing testcases
 - Continuous longer running fuzzing (batch fuzzing) that asynchronously find
    deeper bugs missed during code change fuzzing and build a minimal corpus for
-   use in code change fuzzing.
-- Coverage reports, so users can see which parts of their code is fuzzed.
+   use in code change fuzzing
+- Coverage reports, so users can see which parts of their code is fuzzed
+- Modular funtionality, so you can decide which features you want to use
 
-ClusterFuzzLite is modular, so you can decide which features you want to use.
 
-ClusterFuzzLite uses [libFuzzer] and important sanitizers for finding bugs in
-C/C++ code:
+## Library and Sanitizers
+
+- [libFuzzer] library for coverage-guided testing
 - [AddressSanitizer], for finding memory safety issues.
-- [MemorySanitizer], for finding use of uninitialized memory.
+- [MemorySanitizer], for finding use of uninitialized memory
 - [UndefinedBehaviorSanitizer], for finding undefined behavior (e.g. integer
-  overflows).
+  overflows)
+
+## Getting Started 
 
 If you're new to using libFuzzer and sanitizers, start with the [Overview] for an explanation of terms and the fuzzing process. 
 
-If you're already familiar with using libFuzzer and sanitizers, you're ready to begin fuzzing with ClusterFuzzLite. Using ClusterFuzzLite is simple and requires two major steps:
```

Comment:
I guess we don't need to say there are two steps, but I think it's good to say this upfront. I don't feel super strongly though.

Reply:
I've taken it out here, but changed the navigation pane on the left to say "Step 1: Build Integration" and "Step 2: Running ClusterFuzzLite. That way the two steps will be clear from any page rather than just in this one spot. Do you think that balances it out?

Reply:
OK. This is fine.